portfolio/src/main/webapp/blog.html

```diff
@@ -54,7 +46,7 @@
     </div>
 
     <footer class="footer">
-      <img class="divider" src="images/divider.png" />
+      <img class="divider" src="resources/divider.png" />
```

Comment:
the path should be `images/` not `resources/ `
as stated in the walkthrough

Reply:
done